### PR TITLE
Protobot climber

### LINKED ISSUE #10: Space Addition
**Describe the solution you'd like**
We need a space in `settings.gradle`

### LINKED ISSUE #18: Add Protobot Climber
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by AmnesiaExe**
**What this PR does**  
Add climber

**What does it fix/add?**  
_Reference issues with "Fixes #10" or "Closes #10", where #10 is the number assigned to the issue_
Fixes #18

### CODE REVIEW

File: 4788/src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -70,9 +70,8 @@ void Robot::TeleopInit() {
 }
 void Robot::TeleopPeriodic() {
 	shooter->teleopOnUpdate(dt);
```
Comment (CJBuchel) [inline 4788/src/main/cpp/Robot.cpp:72]: single tab slipped through the gaps. That's fine, we'll fix it in another pr

File: 4788/src/main/include/ControlMap.h [hunk 2]
```diff
@@ -77,5 +78,9 @@ struct ControlMap {
   // Intake
 
   // Climber
+   static constexpr int ClimberPort = 0;
```
Comment (CJBuchel) [inline 4788/src/main/include/ControlMap.h:81]: Also using tabs here instead of spaces

**Review by CJBuchel — APPROVED** [review]

File: 4788/src/main/cpp/Robot.cpp (hunk 1 above)
Comment (AmnesiaExe) [reply 4788/src/main/cpp/Robot.cpp:72]: FIxed

File: 4788/src/main/include/ControlMap.h (hunk 2 above)
Comment (AmnesiaExe) [reply 4788/src/main/include/ControlMap.h:81]: FIxed

**Pull request merged**